src/planscape/planning/services.py

```diff
@@ -1,14 +1,14 @@
 import math
 import os
+import zipfile
+import fiona
```

Comment:
my linter was very vocal about the order here, and I got tired of seeing it complain, so I moved them.